### PR TITLE
Terraform AWS ELB SLO release notes

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

**Pending SME review - do not merge**

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: blog-service/2022-12-02-slo.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - reliability management
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+AWS ELB users can now install Sumo Logic out-of-the-box SLOs and associated monitors using [our new Terraform script](https://github.com/SumoLogic/sumologic-solution-templates/tree/master/slo_packages/aws).
+
+This script allows you to install Sumo Logic SLOs in your specified AWS ELB directory, then configure SLO alert notifications (e.g., latency limit breached, server error limit breached). Once installed, you can view and edit your SLO alerts via Terraform or from your Sumo Logic **Monitors** page.
```
Comment (JainM6) [inline blog-service/2022-12-02-slo.md:16]: I'd remove "**, ** **then**" and rephrase this as:
This script allows you to install Sumo Logic SLOs in your specified AWS ELB directory and configure SLO alert notifications (e.g., latency limit breached, server error limit breached. Once installed, you can view and edit your SLO alerts via Terraform or from your Sumo Logic Monitors page.

**Review by JainM6 — CHANGES_REQUESTED** [review]

File: blog-service/2022-12-02-slo.md (hunk 1 above)
Comment (kimsauce) [reply blog-service/2022-12-02-slo.md:16]: thank you!

Comment (kimsauce) [inline blog-service/2022-12-02-slo.md:16]:
```suggestion
This script allows you to install Sumo Logic SLOs in your specified AWS ELB directory and configure SLO alert notifications (e.g., latency limit breached, server error limit breached). Once installed, you can view and edit your SLO alerts via Terraform or from your Sumo Logic **Monitors** page.
```

Comment (kimsauce) [inline blog-service/2022-12-02-slo.md:16]:
```suggestion
This script allows you to install Sumo Logic SLOs in your specified AWS ELB directory and configure SLO alert notifications (e.g., latency limit breached, server error limit breached). Once installed, you can view and edit your SLO alerts via Terraform or from your Sumo Logic **Monitors** page.
```

**Review by JainM6 — APPROVED** [review]
Looks good.
